lib/events-scraper/utils.rb

```diff
@@ -6,6 +6,7 @@
 WEBSITE = 'http://webmontag.de'
 FACEBOOK_URL = ''
 ROOT_DIR = File.expand_path("../../../", __FILE__)
+EVENTS_DIR = "#{ROOT_DIR}/data/events/"
```

Comment:
the `EVENTS_DIR` constant is not used anywhere else besides the event class, so it does not make sense (yet) to extact it to a general "utils" file